src/js/juju-cards-v1.0.0.js

```diff
@@ -60,6 +60,12 @@ let jujuCards = () => {
     }
 
     let addLink = `${demoDomain}/?deploy-target=${getImageID(id)}`;
+    let deployTitle = 'Deploy with Juju';
+
+    if (typeof(card.dataset.dd) !== 'undefined') {
+      addLink = `${demoDomain}/?dd=${getImageID(id)}`;
+      deployTitle = 'Direct Deploy with JAAS';
```

Comment:
I think having one CTA for two different A's is a bad pattern, long term there will be a decision about going all in with DD or add to canvas but DD isn't yet in a state for wide use. Again I think as DD isn't quite there yet, we should hold off on the documentation.

This branch is for a small controlled intake of DD's so that we can see the stats of those who arrive in the DD flow from a couple of external sources to get a broader picture about how and if it works and to inform future changes to both juju-cards and DD.